doc/source/whats_new.rst

```diff
@@ -137,13 +139,13 @@ API
    - find_events and read_events functions have a new parameter `mask` to set some bits to a don't care state by `Teon Brooks`_
 
    - New channels module including layouts, electrode montages, and neighbor definitions of sensors which deprecates
-    ``mne.layouts`` by `Denis Engemann`_
+    `mne.layouts` by `Denis Engemann`_
```

Comment:
i tried. can't render docs b/c I can't get pygraphviz to install; there was a compilation error :/